### PR TITLE
Create Image upload “content-uploads-_inbox-warning-banner-jpg”

### PR DESCRIPTION
**Pull request by nick-mon1**
Adding new image to replace hard coded content 

[Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cms/image-upload/content-uploads-_inbox-warning-banner-jpg//2024/08/07/it-warning-banners-how-gsa-is-working-to-stop-unnecessarily-frightening-users/)

Testing:
- visit preview and verify that the GSA wanner banner is present and check that with a reduced screen size the image does not break or mess with any wrapping issues

### CODE REVIEW

File: data/images/warning-banner.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  # https://s3.amazonaws.com/digitalgov/warning-banner.png
+  # Image shortcode: {{< img src=warning-banner >}}'
+  date     :  2024-09-13 17:39:43 -0400
+  uid      :  warning-banner
+  width    :  1362
+  height   :  460
+  format   :  png
+  credit   :  
+  caption  :  
+  alt      :
```
Comment (bonnieAcameron) [inline data/images/warning-banner.yml:11]:
```suggestion
  alt      :  A screenshot of a now-defunct warning banner that GSA once used to tell users that the program they were opening was for official use only. There are dozens of asterisks and much of the text is in all-caps, creating an alarming experience what was an every-day message."
```

**Review by bonnieAcameron — CHANGES_REQUESTED** [review]
Looks good, but the image needs alt-text. Suggested some for you to review/commit.

**Comment by clmedders** [comment]
@bonnieAcameron thank you for adding that!

@RileySeaburg this is ready for a final review and merge

**Review by RileySeaburg — APPROVED** [review]
Awesome job, merging because the suggested changes have been addressed.

**Comment by RileySeaburg** [comment]
I can't merge actually until you approve @bonnieAcameron

**Review by bonnieAcameron — APPROVED** [review]

**Pull request closed without merging**